src/components/layout.js

```diff
@@ -14,33 +14,41 @@ import QuickThanks from "./quickthanks"
 import Reactions from './reactions'
 
 
-const Paywall = ({ floating }) => {
+const Paywall = ({ page }) => {
   const copyDiv = useRef(null)
 
-  useLayoutEffect(() => {
-    if (typeof window !== "undefined" && floating) {
-
-      if (
-        !window.localStorage.getItem("unlock_handbook") ||
-        !window.localStorage.getItem("sale_id")
-      ) {
-        const children = [].slice.call(document.getElementById('content').children);
-     
-        let isLocked = false;
-        for (const child of children) {
-          if (child.id === 'lock') isLocked = true;
-          if (isLocked === true ) {
-            child.style.display = 'none';
+
+  useEffect(() => {
+    if (
+      !window.localStorage.getItem("unlock_handbook") ||
+      !window.localStorage.getItem("sale_id")
+    ) {
+      setTimeout(
```

Comment:
window.requestAnimationFrame is better here I think.

How come you can't do this in the same hook as the other paywall stuff?